.github/workflows/rust.yml

```diff
@@ -0,0 +1,22 @@
+name: Rust
+
+on:
+  push:
+    branches: [ flatcar-master ]
+  pull_request:
+    branches: [ flatcar-master ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      run: cargo build --verbose
+    - name: Run tests
+      run: cargo test --verbose
```

Comment:
Ditto verbose? With equivalent of `go test -v`, you should only use `-v` while developing, so it's easy to find what failed.

Reply:
ditto